webrender/src/renderer.rs

```diff
@@ -1306,18 +1306,33 @@ struct TargetSelector {
 #[cfg(feature = "debug_renderer")]
 struct LazyInitializedDebugRenderer {
     debug_renderer: Option<DebugRenderer>,
```

Comment:
cleaner to have `Option<Result<_>>` instead

Reply:
Why would that make things better?

Reply:
because the state expressed with an extra `bool` is less safe and more redundant (e.g. it doesn't prohibit Some() + true combo)